modules/widgets/contact-info.php

```diff
@@ -71,6 +71,12 @@ function widget( $args, $instance ) {
 			if ( $instance['title'] != '' )
 				echo $before_title . $instance['title'] . $after_title;
 
+			/**
+			 * Fires at the beginning of the Contact Info widget, after the title.
+			 *
+			 * @since 3.9.2
+			 */
+			do_action( 'jetpack_contact_info_widget_beginning' );
```

Comment:
Maybe this could be named `_start` instead? Writing `beginning` is prone to errors for non-english devs and there are a couple of actions in core that end in `_start` so it's more consistent.